x-pack/plugin/ent-search/src/test/java/org/elasticsearch/xpack/entsearch/analytics/AnalyticsTemplateRegistryTests.java

```diff
@@ -65,15 +66,14 @@
 
 public class AnalyticsTemplateRegistryTests extends ESTestCase {
```

Comment:
ℹ️ Updated these tests since I realized they were partial (missing checks for composable templates)